.github/workflows/ci-codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL Analysis"
+
+on:
+  push:
+    branches: [develop]
```

Comment:
should be: `branches: [master, develop]`

Reply:
done, we had an issue on steps-mongo when master was included so I had to remove it and that is why I left it out here, but added it back.